Makefile

```diff
@@ -23,8 +23,25 @@ DOCKER_RUN := mkdir -p ../.go-pkg-cache bin $(GOMOD_CACHE) && \
 MAKE_DIRS=$(shell ls -d */)
 
 generate:
-	make -C api gen-files 
-	make -C libcalico-go gen-files
+	$(MAKE) -C api gen-files
+	$(MAKE) -C libcalico-go gen-files
+	$(MAKE) -C felix protobuf
+	rm app-policy/proto/*.pb.go
```

Comment:
I should fix this.   I could add a clean target here and make sure clean deletes the photos from app-policy.   WDYT?